internal/binder/factory.go

```diff
@@ -14,15 +14,28 @@
 
 package binder
 
-import "github.com/lf-edge/ekuiper/pkg/api"
+import (
+	"github.com/lf-edge/ekuiper/internal/plugin"
+	"github.com/lf-edge/ekuiper/pkg/api"
+)
 
 type SourceFactory interface {
 	Source(name string) (api.Source, error)
 	LookupSource(name string) (api.LookupSource, error)
+	// GetSourcePlugin use source type name (mqtt/video) to get source plugin installation information
+	// first return value is extension type, only native/portable plugin type have installation information
+	// second return value is the plugin name
+	// third is the plugin installation information
+	GetSourcePlugin(name string) (plugin.EXTENSION_TYPE, string, string)
```

Comment:
rename to SourcePluginInfo? To be consistent as the other two func names. The same for other factories

Reply:
ack